_includes/common/chefsBtn.html

```diff
@@ -0,0 +1,4 @@
+<div class="text-center mt-5">
```

Comment:
If this is only going to be used the once it's not really needed to be a common include. It's not a specific chefs button too it looks like, just a button with page specific variables, so I'd just put this button on the page where it's needed.

Reply:
good call.